plugins/GSdx/stdafx.cpp

```diff
@@ -71,16 +71,48 @@ void vmfree(void* ptr, size_t size)
 	VirtualFree(ptr, 0, MEM_RELEASE);
 }
 
+HANDLE fh = NULL;
+uint8* Next[4];
+
 void* fifo_alloc(size_t size, size_t repeat)
 {
-	// FIXME check linux code
-	return vmalloc(size * repeat, false);
+	ASSERT(fh == 0);
+
+	const char *file_name = "/GSDX.mem";
+	fh = CreateFileMapping(INVALID_HANDLE_VALUE, NULL, PAGE_READWRITE, 0, repeat * size, file_name);
```

Comment:
Is `file_name` necessary? Couldn't nullptr just be passed in so the mapping is created without a name?